packages/react/src/components/Paper/Paper.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import Paper from './Paper.tsx';
+
+export const meta = {
+  component: Paper,
+  title: StoryConfig.Paper.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => (
+  <Box
+    sx={{
+      display: 'flex',
+      flexWrap: 'wrap',
+      '& > :not(style)': {
+        m: 1,
+        width: 128,
+        height: 128,
+      },
+    }}
+  >
+    <Paper {...args} />
+  </Box>
+)
+
+# Paper
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+In Material Design, the physical properties of paper are translated to the screen.
```

Comment:
We might need to update the text to match with oxygen UI I guess

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/156/commits/0f9fa5b6e2dd61aa147c7aa60e68bfd1c97ec6be